backend/src/authentication/verifyToken.js

```diff
@@ -0,0 +1,40 @@
+const jwt = require('jsonwebtoken')
```

Comment:
could we move this file to a src/services folder instead of authentication folder

Reply:
sure. I will do that